webapp/components/channel_invite_modal.jsx

```diff
@@ -30,8 +30,8 @@ export default class ChannelInviteModal extends React.Component {
         this.handleInviteError = this.handleInviteError.bind(this);
         this.search = this.search.bind(this);
 
-        this.page = 0;
         this.term = '';
+        this.page = 0;
```

Comment:
They're not used in the rendering, they're just workarounds. I wanted to keep as much of that logic in the lower component as possible but some cases like this one needed the information above.